### PR TITLE
Add manifest ability specs.

### PR DESCRIPTION
**Pull request by tpendragon**

### CODE REVIEW

**Comment by tpendragon** [comment]
@escowles Can you review? It's complaining about coverage, but only because total lines went down.

File: spec/ability/curation_concern_creator_spec.rb [hunk 1]
```diff
@@ -59,11 +62,14 @@
       it { should_not be_able_to(:create, ScannedResource.new) }
       it { should_not be_able_to(:create, FileSet.new) }
       it { should_not be_able_to(:read, campus_only_scanned_resource) }
+      it { should_not be_able_to(:manifest, campus_only_scanned_resource) }
       it { should_not be_able_to(:pdf, campus_only_scanned_resource) }
       it { should_not be_able_to(:update, scanned_resource) }
       it { should_not be_able_to(:destroy, scanned_resource) }
+      it { should_not be_able_to(:manifest, campus_only_scanned_resource) }
```
Comment (escowles) [inline spec/ability/curation_concern_creator_spec.rb]: Should be just `scanned_resource` right?

**Comment by escowles** [comment]
Looks good other than that one small thing.  Also, add #139 to just add coverage for those two lines.

**Comment by tpendragon** [comment]
Working on doing this without persisting those resources..

**Pull request merged**